### PR TITLE
Raghavdev

### PR DESCRIPTION
**Pull request by itsmeraghav**
update api

### CODE REVIEW

File: lib/models/models/Industry.model.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      type: Number,
+      default: 1,
+    },
+
+    is_deleted: {
+      type: Number,
+      default: 0,
+    },
+
+    created_by: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```
Comment (electroit) [inline lib/models/models/Industry.model.js:28]: We don't require creator

File: lib/models/models/Joblevel.model.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      type: Number,
+      default: 1,
+    },
+
+    is_deleted: {
+      type: Number,
+      default: 0,
+    },
+
+    created_by: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```
Comment (electroit) [inline lib/models/models/Joblevel.model.js:28]: We don't require creator

File: lib/models/models/Jobtype.model.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      type: Number,
+      default: 1,
+    },
+
+    is_deleted: {
+      type: Number,
+      default: 0,
+    },
+
+    created_by: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```
Comment (electroit) [inline lib/models/models/Jobtype.model.js:28]: We don't require creator

File: lib/models/models/Skills.model.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      type: Number,
+      default: 1,
+    },
+
+    is_deleted: {
+      type: Number,
+      default: 0,
+    },
+
+    created_by: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```
Comment (electroit) [inline lib/models/models/Skills.model.js:28]: We don't require creator

File: src/api/routes/district/DistrictController.js [hunk 5]
```diff
@@ -87,7 +82,7 @@ class UserController {
           recordsTotal:
             results && results.records_total ? results.records_total : 0,
         };
-        return res.success(data, req.__("DISTRICT_SETTING_INFORMATION"));
+        return res.success(data, req.__("DISTRICT_LIST_GENRETED"));
```
Comment (electroit) [inline src/api/routes/district/DistrictController.js:85]: spelling check

**Comment by itsmeraghav** [comment]
DONE

**Pull request merged**